radicle-cli/src/commands/assign.rs

```diff
@@ -5,58 +5,58 @@ use anyhow::anyhow;
 use radicle::prelude::Did;
 
 use crate::terminal as term;
-use crate::terminal::args;
+use crate::terminal::args::{Args, Error, Help};
 use radicle::cob;
 use radicle::cob::issue;
 use radicle::storage::WriteStorage;
 
-pub const HELP: args::Help = args::Help {
+pub const HELP: Help = Help {
     name: "assign",
-    description: "assign an issue",
+    description: "Assign an issue",
     version: env!("CARGO_PKG_VERSION"),
     usage: r#"
 Usage
 
-    rad assign <issue> <did>
+    rad assign <issue> --to <did>
 
 Options
 
-    --help      Print help
+    --to <did>    Assignee to add to the issue
+    --help        Print help
 "#,
 };
 
 #[derive(Debug)]
 pub struct Options {
     pub id: issue::IssueId,
-    pub peer: Did,
+    pub to: Did,
```

Comment:
Should we allow `Vec<Did>` while we're at it?

Reply:
Actually I was thinking multiple issues would be more useful to start, eg. assigning someone to multiple issues: `rad assign id1 id2 id3 --to finto`

Reply:
As usually we only have one person assigned per issue. But yeah, at the COB level what you proposed is "free".

Reply:
Ya, I presumed the assigning can have multiple people since the code was using `[]`